content/blog/2023/2023-08-28-fairness-adversarial-debiasing.md

```diff
@@ -0,0 +1,59 @@
++++
+author = "Žan Mervič"
+date = "2023-08-28"
+draft = false
+title = "Orange Fairness - Adversarial Debiasing"
+type = "blog"
+thumbImage = "/blog_img/2023/2023-08-28-fairness-adversarial-debiasing-thumb.png"
+frontPageImage = "/blog_img/2023/2023-08-28-fairness-adversarial-debiasing-thumb.png"
+blog = ["fairness", "adversarial debiasing"]
+shortExcerpt = "Learn how to use the Adversarial Debiasing model in Orange for fairer machine learning."
+longExcerpt = "This blog post focuses on the Adversarial Debiasing model in Orange, a tool for enhancing fairness in your machine learning algorithms. We will walk through how to use it and explain the trade-offs that come with using fairness algorithms."
++++
+
+In the [previous blog post]({{< ref "2023-08-25-fairness-reweighing-preprocessor.md" >}}), we talked about how to use the Reweighing widget as a preprocessor for a model. This blog post will discuss the Adversarial Debiasing model, a bias-aware model. We will also show how to use it in Orange.
+
+### Adversarial Debiasing:
+
+[Adversarial Debiasing](https://arxiv.org/abs/1801.07593) is an in-processing type of fairness mitigation algorithm. It is a technique that uses adversarial training to mitigate bias. It involves simultaneous training of a predictor and a discriminator. The goal of the predictor is to predict the target variable accurately. At the same time, the discriminator aims to predict the protected variable (such as gender or race) based on the predictor's predictions. The main goal is to maximize the predictor's ability to predict the target variable while reducing the discriminator's ability to predict the protected variable based on those predictions. Because of the Adversarial Debiasing implementation in the [AIF360](https://aif360.res.ibm.com/) package we are using, the algorithm focuses on Disparate Impact and Statistical Parity Difference fairness metrics.
+
+Like most model widgets in Orange, the Adversarial Debiasing widget allows users to tune the model's hyperparameters to achieve the best possible results. The widget's interface is similar to other model widgets, allowing the user to set the model's hyperparameters. The widget's interface is shown in the image below.
```

Comment:
"Like most model widgets in Orange, the Adversarial Debiasing widget allows users to tune the model's hyperparameters to achieve the best possible results" -> delete.

Reply:
Actually, delete the whole paragraphs but the last sentence. What would you gain by so much generic speak? If you'd want to describe some parameters, fine, but just writing that some exist is useless.